dozer-cli/src/pipeline/connector_source.rs

```diff
@@ -314,23 +313,23 @@ impl Source for ConnectorSource {
                         .get(&schema_id)
                         .ok_or(ConnectorSourceError::PortError(schema_id))?;
 
-                    let labels = [
+                    let mut labels = vec![
                         ("connection", self.connection_name.clone()),
                         ("table", table_name.to_string()),
                     ];
                     match &kind {
                         IngestionMessageKind::OperationEvent(Operation::Delete { .. }) => {
-                            increment_counter!(SOURCE_DELETE_COUNTER_NAME, &labels);
+                            labels.push(("operation_type", "delete".to_owned()))
```

Comment:
We don't need these `to_owned`

Reply:
since the labels have type `Vec<(&str, String)>` we have to convert it anyway

Reply:
Sure my bad.